itests/test-jdbc/src/test/resources/custom_hosts_file

```diff
@@ -0,0 +1,3 @@
+127.0.0.1       localhost
+127.0.0.1       test-standalone-jdbc-plain
+127.0.0.1       test-standalone-jdbc-kerberos
```

Comment:
Does the JDBC JAR relate to the authentication mode? If there is no relationship, can we keep only the simplest plain mode to simplify the code?

Reply:
Do we need to run the test in the kerberos mode?  Is the plain model not sufficient?